protected/migrations/m190408_065948_create_contribution_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Are there any reasons why the `down()` functions are not implemented in these database migration scripts?